src/celeritas/global/ActionInterface.hh

```diff
@@ -44,7 +45,7 @@ class CoreState;
  *
  * The description should be a verb phrase (lowercase start).
  */
-class ActionInterface
+class ActionInterface : public Interface
```

Comment:
Now anyone can use `Interface* derp = new MyAction; derp = new MyOutputInterface;`. Definitely shouldn't be public, probably should be templated.